src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,15 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 0) {
```

Comment:
the same problem
![image](https://user-images.githubusercontent.com/36365171/70029877-b565da80-15b0-11ea-9713-b8b588332349.png)